README.md

```diff
@@ -143,17 +143,20 @@ There are several command line options to start the daemon:
 Currently you would likely want to start the daemon like this:
 
 ```bash
-cargo run -- -r http://knilxof.org:4242 --disable-tls
+./run.sh -- -r http://knilxof.org:4242 --disable-tls
```

Comment:
I discussed with @JohanLorenzo and we decided to keep it especially to be able to use cargo's `--verbose`.

We could special-case this but this is more work.